app/core/cclw_ingestion/family.py

```diff
@@ -2,10 +2,10 @@
 
 from sqlalchemy.orm import Session
 
-from app.core.ingestion.ingest_row import DocumentIngestRow
-from app.core.ingestion.metadata import add_metadata
+from app.core.cclw_ingestion.ingest_row_cclw import CCLWDocumentIngestRow
+from app.core.cclw_ingestion.metadata import add_metadata
```

Comment:
could these not live in a sub-module? I'm just worried what this will look like when we have many sources... `app.core.ingestion.cclw.ingest_row` would work, no?

Reply:
lol ... yes ... another thing on my list ... but want to get to a GREEN state first :smile: